scripts/ethbinaries.sh

```diff
@@ -150,6 +150,14 @@ elif [[ "$OSTYPE" == "darwin"* ]]; then
 
 	if [[ $DEV_TEST -eq 1 ]]; then
 		echo "ETHBINARIES - INFO: Building MacOSX for development test.";
+		cmake .. -DCMAKE_BUILD_TYPE=RelWithDebInfo
+		make -j4
+		if [[ $? -ne 0 ]]; then
+			echo "ETHBINARIES - ERROR: Building for Macosx failed.";
+			exit 1
+		fi
+	else
+		echo "ETHBINARIES - INFO: Building MacOSX binaries.";
 		# Temporarily forcing OS X binaries to Debug as a workaround for the Heisenbug.
```

Comment:
I did.   See https://github.com/ethereum/webthree-helpers/blob/develop/scripts/ethbinaries.sh.

Github's diff just looks strange because of similarities between the two blocks of code.